package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@ronas-it/axios-api-client",
+  "version": "0.1.0",
+  "description": "Wrapper utilities for interaction with REST APIs using Axios",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup-node",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
+    "format": "prettier --write .",
```

Let's update these scripts to match our other projects:

```suggestion
    "lint": "npx tsc && npx eslint ./",
    "format": "npx prettier --write . && npm run lint -- --fix",
```